.github/workflows/prepare-release-proposal.yml

```diff
@@ -0,0 +1,100 @@
+name: Prepare release proposal
+
+on:
+  workflow_dispatch:
+    inputs:
+      base-branch:
+        description: 'Release branch'
+        required: true
+        type: choice
+        options:
+        - v4.x
+        - v5.x
+      release-type:
+        description: 'Release type'
+        required: true
+        type: choice
+        options:
+        - minor
+        - patch
+
+jobs:
+  create-proposal:
+    runs-on: ubuntu-latest
+
+    permissions: write-all
+
+    steps:
```

Comment:
Maybe most of the steps, but we need to have `Create PR` step in a separate step before the package.json update, because it needs different GH access token

Reply:
I guess I meant more for whatever can be done locally, like checking out the right release line, creating the proposal branch with the right name, merging commits, committing the new version, etc.

Reply:
Most of this is already extracted to a script, so that's fine for now and definitely not a blocker.